app/src/main/res/navigation/nav_graph_onboarding.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<navigation xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/nav_graph_onboarding"
+    app:startDestination="@id/navigation_intro">
+
+    <fragment
+        android:id="@+id/loginFragment"
+        android:name="com.internship.move.LoginFragment"
+        android:label="LoginFragment">
+        <action
```

Comment:
leave an empty line between xml tags and between the parent tag and first child tag (similarly to how you'd do for a fragment xml)
Periodically format your code, especially if you add these using the Android Studio tool